### PR TITLE
Merge the tutorials under tutorials->clusters into tutorials->security

### LINKED ISSUE #30778: Merge tutorials->clusters into tutorials->security
**This is a Feature Request**

**What would you like to be added**

Post-merge of #30422 , move the tutorials under https://github.com/kubernetes/website/tree/main/content/en/docs/tutorials/clusters into https://github.com/kubernetes/website/tree/main/content/en/docs/tutorials/security

Merge clusters section into security section here: https://github.com/kubernetes/website/blob/main/content/en/docs/tutorials/_index.md

**Why is this needed**
https://github.com/kubernetes/website/pull/30422#pullrequestreview-823841225

Issue description is a slight modification from original request in the comment

**Comments**

/hold for #30422 
/cc @sftim 
/sig docs security

### PR DESCRIPTION
**Pull request by Babapool**
Fixes #30778 

It moves the move the tutorials under https://github.com/kubernetes/website/tree/main/content/en/docs/tutorials/clusters into https://github.com/kubernetes/website/tree/main/content/en/docs/tutorials/security

The Clusters section is merged into the Security section here: https://github.com/kubernetes/website/blob/main/content/en/docs/tutorials/_index.md 
Based on the suggestion made in the PR comment : https://github.com/kubernetes/website/pull/30422#pullrequestreview-823841225

Also redirects of the tutorial has been made to their new location ( Since this is my first PR working with `redirects` and involves `moving of files`, appreciate all the feedback)

Signed by : Vitthal Sai vitthalsai2001@gmail.com

### CODE REVIEW

**Comment by Babapool** [comment]
Hi @shannonxtreme @daminisatya can you PTAL at this PR when you are free. Thanks

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-31014--kubernetes-io-main-staging.netlify.app/docs/tutorials/

Redirects in the preview are working

/lgtm

**Comment by annajung** [comment]
/lgtm
/sig security

leaving approval for @sftim

File: static/_redirects [hunk 1]
```diff
@@ -546,3 +546,6 @@
 /id/docs/concepts/configuration/pod-overhead/     /id/docs/concepts/scheduling-eviction/pod-overhead/   301
 
 /pt/*         /pt-br/:splat  302!
+
+/docs/tutorials/clusters/apparmor/     /docs/tutorials/security/apparmor/   301
+/docs/tutorials/clusters/seccomp/      /docs/tutorials/security/seccomp/    301
```
Comment (sftim) [inline static/_redirects:551]: nit: I'd usually put this before the localization-specific redirects.

**Review by sftim — APPROVED** [review]
/approve

**Comment by sftim** [comment]
Oh, merge conflict?

@Babapool would you like help fixing that?

**Comment by Babapool** [comment]
Yes please, I would definitely love the help in fixing this merge conflict. Thank you @sftim. (Also will you be able to tell me more about this conflict because when I opened the PR GitHub told me it can be  merged, thanks.)

**Comment by Babapool** [comment]
Also I can push the nit changes only after the merge conflict is resolved?

**Comment by annajung** [comment]
Hi @Babapool, merge conflicts happen when another contributor changes to the same file/ line of a file. In this case, looks like conflict may be in the `static/_redirects` file.

Try following the steps outlined in the [merge conflict and rebasing](https://kubernetes.io/docs/contribute/new-content/open-a-pr/#merge-conflicts-and-rebasing) and if you have any questions, feel free to reach out in #sig-docs slack channel

**Comment by Babapool** [comment]
Hi @annajung, I will try to follow the steps outline and in case of any doubts I will surely reach out in the slack channel. Thanks

**Comment by Babapool** [comment]
Hi @annajung @sftim, I think I may have messed (my first time doing a rebase). If that is the case, please suggest what can I do next. Thanks

**Comment by sftim** [comment]
With your permission @Babapool I can fix this one up for you and push the squashed change to your fork.

https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits covers how to do a squash; for learning how to rebase in general, I'd look to the wider internet for a guide / book / video that fits your level of Git experience.

**Comment by Babapool** [comment]
@sftim Thank you, it would be really great if you could handle this one for me. I'll read up on this more so that I can handle such situations in a better manner next time. Thanks again!

**Comment by sftim** [comment]
Try that!

**Comment by Babapool** [comment]
Hi @sftim, Thank you once again for handling this one for me. Hope to handle such cases better next time.

**Pull request merged**